src/Server.UI/Pages/Analytics/OutcomeQualityDipSample/DipSample.razor.cs

```diff
@@ -96,101 +97,74 @@ private async Task OnExport()
         }
     }
 
-    private async Task OnReview()
+    private Task OnReview() => PerformAction(
+        new ReviewDipSampleOutcome.Command() { SampleId = SampleId },
+        "Are you sure you are ready to mark this sample as reviewed? This cannot be undone.",
+        "Mark Reviewed",
+        "Review Submitted"
```

Comment:
```suggestion
        "Review Sample",
        "Review submitted"
```

Nitpick: Make the dialog title and confirmation message consistent with the others